acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +82,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
-          step "assert no redacted data in syslog" do
-            #sigh.  gee, thanks for the help, beaker!
-            result = dump_puppet_log(agent)
```

Comment:
it probably needs an `else logger.warn "we're not testing logs on #{platform}"` or something, but i wouldn't hold up this PR on that